src/docs/patterns/navigation.mdx

```diff
@@ -1,70 +1,311 @@
+        react={'https://pxblue-components.github.io/react/?path=/story/components-drawer--in-drawer-layout'}
+    />
+    <MaterialDesignDescription
+        title={'Navigation Drawer '}
+        url={'https://material.io/components/navigation-drawer/'}
+        style={{ marginRight: 0 }}
+    />
+</React.Fragment>
+
+## Permanent
+
+A permanent drawer is always visible and always open. The permanent drawer is typically full-width, as opposed to the rail variant (see below). Use this variation if you have:
+
+-   five or more top-level pages
+-   a nested navigation hierarchy
+-   enough space to justify it (typically desktop-only applications)
+
+<ImageGrid images={[NavigationPermanentImg]} caption={['A permanent drawer on desktop.']} regularWidth />
+
+## Rail
+
+A navigation rail is a permanently collapsed drawer. The rail can show less information than a standard drawer but has the advantage of a more compact, space-saving package. Use this variation if you have:
+
+-   fewer than seven top-level pages
+-   a flat navigation hierarchy
+-   limited space - recommended for HMI devices
+
+<ImageGrid images={[NavigationRailImg]} 
+caption={[
+        'Condensed Rail on desktop.',
+    ]}
+    regularWidth 
 />
-<MaterialDesignDescription
-    title={'Navigation Drawer '}
-    url={'https://material.io/components/navigation-drawer/'}
-    style={{marginRight: 0}}
+<ImageGrid images={[NavigationRailLabelsImg]} 
+caption={[
+        'Standard rail with labels on desktop.',
+    ]}
+    regularWidth 
 />
 
-## Fixed Navigation
+## Persistent
 
-The most basic navigation panel is fixed in place and always visible on the left side of the application. Note: a fixed navigation panel is not suitable when going to smaller resolutions or mobile devices.
+A persistent drawer is an always-visible drawer that can be opened and closed (expanded or collapsed). In its open state, it looks and behaves like a permanent drawer. In its closed state, it looks and behaves like a navigation rail. The drawer is typically opened and closed by clicking an icon in the header. It can also be opened momentarily on desktop applications by hovering with the mouse. Use this variation if you have:
 
-## Collapsible Navigation
+-   fewer than seven top-level pages
+-   primary pages with labels that require complex icons or too long to wrap effectively
+-   limited space but require a nested navigation hierarchy
 
-A more common implementation of the navigation panel is a collapsible panel that can be opened and closed via the menu icon in the top left corner of the application. This implementation can have two different variations: content-resize or content-cover.
+<React.Fragment>
+    <ImageGrid
+        images={[CollapsibleImg]}
+        caption={['1. Displays the normal collapsed state of the menu.', '2. Displays the open state of the menu.']}
+        regularWidth
+    />
+    <ImageGrid
```

Comment:
Nice gif...do we want to speed it up to real-time (300ms i think?), or is it slow-mo to better show the interaction?

Reply:
I personally think slow motion shows the transition better.